src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2283,20 +2283,20 @@ public override void Accept(OperationVisitor visitor)
     /// </remarks>
     internal sealed partial class VariableDeclaration : Operation, IVariableDeclaration
     {
-        public VariableDeclaration(ILocalSymbol variable, IOperation initialValue, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+        public VariableDeclaration(ImmutableArray<ILocalSymbol> variables, IOperation initialValue, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
```

Comment:
Did you edit this file directly?  Or was this created by running a generator?  If so, where is the original file this was generated from?